lib/content-services/document-list/components/document-list.component.ts

```diff
@@ -612,12 +619,12 @@ export class DocumentListComponent implements OnInit, OnChanges, AfterContentIni
                         pagination: result.list.pagination
                     }
                 };
-                this.onPageLoaded(page);
+                this.onPageLoaded(page, merge);
             })
             .catch(error => this.error.emit(error));
     }
 
-    private loadRecent(): void {
+    private loadRecent(merge: boolean = false): void {
```

Comment:
please avoid use of flag arguments  for more informarion
https://martinfowler.com/bliki/FlagArgument.html

Reply:
@eromano the merge comes from the "infinite load" feature, we already have `reload(merge: boolean = false)` and `loadFolder(merge: boolean = false)`

Reply:
As mentioned by Denys, I used the 'merge' to assure that the infinite load will work also on the custom source lists.